ratis-examples/src/test/java/org/apache/ratis/examples/filestore/FileStoreStreamingBaseTest.java

```diff
@@ -136,7 +136,7 @@ private void testMultipleFiles(String pathBase, int numFile, SizeInBytes fileLen
       writerFutures.add(executor.submit(callable));
     }
     for (Future<FileStoreWriter> future : writerFutures) {
-      future.get();
+      future.get().close();
```

Comment:
This is a good catch!  A new client is created for each write, we should close them.

Reply:
@szetszwo Thank you very much for reviewing the code!